src/sql/scales_distribution_for_expertise.sql

```diff
@@ -0,0 +1,29 @@
+WITH latest_expertise_log AS (
```

Comment:
@tkellen any optimizations you can offer for this query would be great.

I'm trying to get one row back with 2 columns:
* `interest`: a 2d array of interest ranking and user count, sorted by ranking
* `experience`: a 2d array of experience ranking and user count, sorted by ranking

And if there are no matching interest or experience ranking levels for the given log entry, that ranking level should NOT be skipped, so we always get counts for 1-5 (even if the count is 0)